<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>WirelessBase</class>
 <widget class="QWidget" name="WirelessBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>3</number>
     </property>
     <item>
      <widget class="QToolButton" name="newButton" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="delButton" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="netSelector" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="advanced_frame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>5</number>
      </property>
      <item>
       <widget class="QComboBox" name="mode" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text" >
          <string>Ad-Hoc</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Managed</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Master</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="essid_label" >
        <property name="text" >
         <string>Network name (ESSID):</string>
        </property>
        <property name="buddy" >
         <cstring>essid</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="essid" />
      </item>
      <item>
       <widget class="QLabel" name="ap_label" >
        <property name="text" >
         <string>Access Point:</string>
        </property>
        <property name="buddy" >
         <cstring>ap</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ap" />
      </item>
      <item>
       <widget class="QLabel" name="nickname_label" >
        <property name="text" >
         <string>Nickname:</string>
        </property>
        <property name="buddy" >
         <cstring>nickname</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nickname" />
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="channel_label" >
          <property name="text" >
           <string>Channel:</string>
          </property>
          <property name="buddy" >
           <cstring>channel</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channel" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="buttonSymbols" >
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="specialValueText" >
           <string>auto</string>
          </property>
          <property name="maximum" >
           <number>14</number>
          </property>
          <property name="minimum" >
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="bitrate_label" >
          <property name="text" >
           <string>Bit Rate:</string>
          </property>
          <property name="buddy" >
           <cstring>bitrate</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="bitrate" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text" >
            <string>auto</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>54</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>48</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>36</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>24</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>18</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>5.5</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>1</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>535</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>netSelector</tabstop>
  <tabstop>mode</tabstop>
  <tabstop>essid</tabstop>
  <tabstop>ap</tabstop>
  <tabstop>nickname</tabstop>
  <tabstop>channel</tabstop>
  <tabstop>bitrate</tabstop>
  <tabstop>newButton</tabstop>
  <tabstop>delButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
